test/unit/org/apache/cassandra/utils/CassandraVersionTest.java

```diff
@@ -302,4 +302,19 @@ private static CassandraVersion version(String str)
         }
         throw new NoSuchMethodException(CassandraVersion.class + "." + name + Arrays.toString(args));
     }
+
+    @Test
+    public void testVersionComparison()
```

Comment:
Should we not add a test describing our expectations including `4.0.700-alpha3` ?